converters/fromZigbee.js

```diff
@@ -1760,6 +1760,23 @@ const converters = {
             };
         },
     },
+    ZHAC_4257050_power: {
```

Comment:
Instead of creating a new converter, can you try using `fz,generic_power`, for that you also need to have the following in `configure`: https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices.js#L1525

Reply:
Ah, gotcha! I will work on this over the next couple of days and get back to you.

Reply:
Thanks for looking at this :+1:

Reply:
Wow, this is cool... it's working great, and I'm actually able to switch another device (that has a custom power converter function) to the new generic one. Is it okay if I include the changes for the other device (switching it to the new generic converter and reading the values) in this PR, or would you like a separate PR for them?

Reply:
Awesome! :trophy:  lets include it in this pr.

Reply:
👍 changes incoming!